README.md

```diff
@@ -1,16 +1,21 @@
 This repository contains a few silly or less silly Babetrace 2 plugins.
 
  * Very simple
-   [source and sink components](https://github.com/simark/babeltrace-fun-plugins/tree/master/my-first-components),
+   [source and sink components](/my-first-components),
```

Comment:
Just wondering, in which situation is that useful.

Could you submit these README fixes as its own PR?

Reply:
If the original and forked repository diverge significantly, it is better that the README links point to the files of the forked repository.

I can submit them as a separate pull request, no problem.